### PR TITLE
Change Chart source for argo workflows and adapt values

### PR DESCRIPTION
**Pull request by enocquet**
### Change Helm Chart source for argo-workflows

- Now using Bitnami Helm Chart for argo-workflows
- Adapted values and version for this new source

### CODE REVIEW

File: create-argo/values.yaml [hunk 1]
```diff
@@ -1,419 +1,271 @@
...
+    path: /metrics
+    serviceMonitor:
+      enabled: true
+  telemetry:
+    enabled: false
+    path: /telemetry
+  workflowWorkers: 32
+  workflowNamespaces:
+    - default
+  workflowDefaults: {}
```
Comment (sjoubert) [inline create-argo/values.yaml:198]: The `podGC` (and others values) should be configured here, we lost that from the previous config.

File: create-argo/values.yaml [hunk 2]
```diff
@@ -1,419 +1,271 @@
...
-    pullPolicy: ""
-  resources: {}
-  args: []
-  env:
-  - name: RESOURCE_STATE_CHECK_INTERVAL
-    value: 1s
-  - name: WAIT_CONTAINER_STATUS_CHECK_INTERVAL
-    value: 1s
-  securityContext: {}
-%{ if USE_MINIO_STORAGE }
```
Comment (sjoubert) [inline create-argo/values.yaml:29]: We lost the s3/artifactRepository config for v3.x compatibility. Unless we are prepared to drop support for old version this is an issue. I'm not 100% sure how the new chart config works but this probably needs to be moved to the `controller.config` entry.

**Review by sjoubert — CHANGES_REQUESTED** [review]
I think a lot of the previous config entries that don't have a direct match in the new chart are config value that can be placed in `controller.config` I just hope we did not miss more than `artifactRepository` and `s3`

File: create-argo/values.yaml [hunk 3]
```diff
@@ -1,419 +1,271 @@
...
+      archive: {{ include "common.tplvalues.render" (dict "value" .Values.controller.persistence.archive.enabled "context" $) }}
+      {{- if or .Values.postgresql.enabled (and .Values.externalDatabase.enabled (eq .Values.externalDatabase.type "postgresql")) }}
+      postgresql:
+      {{- else if or .Values.mysql.enabled (and .Values.externalDatabase.enabled (eq .Values.externalDatabase.type "mysql")) }}
+      mysql:
+      {{- end }}
+        host: {{ include "argo-workflows.controller.database.host" . }}
+        port: {{ include "argo-workflows.controller.database.port" . }}
+        database: {{ include "argo-workflows.controller.database" . }}
+        tableName: argo_workflows
```
Comment (sjoubert) [inline create-argo/values.yaml:167]: As seen together this should be `workflows`

Comment (enocquet) [reply create-argo/values.yaml]: updated

File: create-argo/values.yaml (hunk 1 above)
Comment (enocquet) [reply create-argo/values.yaml]: Updated

**Review by sjoubert — APPROVED** [review]

File: create-argo/values.yaml (hunk 2 above)
Comment (enocquet) [reply create-argo/values.yaml:29]: Updated too

**Pull request merged**